### DISCUSSION TITLE
plot_model chart size and export to dataframe

### DISCUSSION THREAD

**Discussion by maragato2000** [pycaret/pycaret#2140] [Q&A] [answered]
I am plotting -plot_model()- a classification report with 40 features and the charts comes flattened (see image attached); It's also impossible to read the confusion matrix. I have tried changing the scale property and exporting as a file with no luck.

1.- Is there a way to tune the chart size to make it readable ?
2.- Is there a way to show them as a dataframe ?

Thanks,
![image](https://user-images.githubusercontent.com/49998387/152855892-731eb4f7-4519-4a6e-979d-cfffacf43cd8.png)

**Comment by maragato2000** [comment, marked as answer]
I found the answer ... just need to add  the following inside the function parameters :

 , plot_kwargs = {'size':(1080, 1320)}

Next step how to pass it to a df ...

**Reply by Rishabvanigotha123** [reply]
how to pass it to dataframe or dictionary? i want to store confusion matrix in a dictionary or a dataframe how to do that?
